src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -111,9 +120,19 @@ flex-cli search set --key category --type enum --scope public -m my-test-marketp
 flex-cli search set --key amenities --type multi-enum --scope public -m my-test-marketplace
 ```
 
+Note that we don't need to supply `--schema-for` option when we add schema for `listing` as `listing` is the default.
+
+## Adding user search schema
```

Comment:
Maybe it would be nice to add a bit of intro on how user profile schema is useful:

"
User profile search schema can be useful, if you have an Integration API application that needs to query different sets of users, depending on some value in the user profile's extended data. For instance, if users have `age` attribute stored in their protected data, you can use the [/users/query endpoint in the Integration API](/references/extended-data/) to find users of a certain age range.

Search schema for user profiles can be added as follows:
"

Reply:
Nice suggestion, added the intro as suggested. :)